src/settings/RoutingAddress/RoutingAddress.js

```diff
@@ -17,7 +17,21 @@ import css from '../ConfigManagerForm.css';
 export const RoutingAddress = ({ label }) => {
   const showCallout = useShowCallout();
   const { addressTypes } = useUserAddressTypes();
-  const { data, isFetching, refetch } = useRoutingAddressSettings();
+  const { data, isFetching, refetch, error } = useRoutingAddressSettings();
+
+  useEffect(() => {
```

Comment:
1. why do you need this?
2. it should be done via onError instead of useEffect
3. please add loading page

Reply:
updated